src/app/pages/maintenance/maintenance.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="container vertical-center montserrat">
+    <div class="row justify-content-center m-2 ">
+      <h1 id="maintenance-heading">Our database is currently under maintenance. While you can view your virtrolio as
+        usual, sending messages will be unavailable for the time being.</h1>
+      <h2 id="maintenance-subheading">We apologize for the inconvenience.</h2>
+    </div>
+    <div class="row justify-content-center mx-3 mt-3">
+      <img src="../../../assets/images/InvalidLink.png" alt="black cloud graphic" id="graphic">
+    </div>
+    <div class="row justify-content-center m-2">
+      <button class="btn-lg btn-primary mt-5 mb-5 mx-3" routerLink="/your-virtrolio">Your virtrolio</button>
+      <button class="btn-lg btn-primary mt-5 mb-5 mx-3" routerLink="/">Back to Home</button>
```

Comment:
Condense margin selectors
```suggestion
      <button class="btn-lg btn-primary my-5 mx-3" routerLink="/your-virtrolio">Your virtrolio</button>
      <button class="btn-lg btn-primary my-5 mx-3" routerLink="/">Back to Home</button>
```